tests/Module/Fixture/DisplayUserEvents.php

```diff
@@ -85,6 +85,67 @@ public function saveUserEvents(int $owner, array $events)
 
         return ['events' => $events, 'eventsId' => $eventsId];
     }
+
+    public function haveIncrementalEvents(int $owner)
+    {
+        $events = ['incremental_event_first', 'incremental_events_second'];
```

Comment:
Don't hardcode it. It's an antipattern. You have to pass the event names for better re-use